src/t_zset.c

```diff
@@ -1697,7 +1697,7 @@ void zsetTypeRandomElement(robj *zsetobj, unsigned long zsetsize, ziplistEntry *
             }
         }
     } else {
-        serverPanic("Unknown zset encoding");
+        serverPanic("Unknown sorted set encoding");
```

Comment:
zset is an acceptable term in redis (even in user facing text, docs, error messages, so certainly in panics).
i rather not change lines if not needed (useless merge conflicts and blame log damage)